QtlConnpassAPI/Model/Events.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  EventModel.swift
+//  ConnpassShowList
+//
+//  Created by 柿崎逸 on 2023/03/26.
+//
+
+import Foundation
+
+struct Events: Decodable, Identifiable {
+    var id = UUID()
+    var title: String
+    var catchCopy: String
+    var eventURL: String
```

Comment:
```suggestion
    let id = UUID()
    let title: String
    let catchCopy: String
    let eventURL: String
```

[nits] letで問題ないと思うので変更したほうが良いかなと

Reply:
commit: be288cd5bcd0b1de10c61c5f3620f3bbda7dad51
letで問題ありませんでした！